.github/workflows/release.yaml

```
@@ -204,12 +224,20 @@ jobs:
 
         make build-plugin-image push-plugin-image
 
+        make build-operator push-operator
+
     - name: Configure git
       run: |
         git config user.email 'kiali-dev@googlegroups.com'
 
         git config user.name 'kiali-bot'
 
+    - name: Create new version bundle (OLM)
+      run: |
+        ./operator/manifests/create-new-version.sh --new-version ${RELEASE_VERSION:1} --channels candidate
```

In order to use this `create-new-version.sh` you need to first download the operator SDK. See: https://github.com/kiali/openshift-servicemesh-plugin/blob/main/operator/manifests/create-new-version.sh#L42

So before this line you need to do `make get-operator-sdk`.
```suggestion
        make get-operator-sdk

        ./operator/manifests/create-new-version.sh --new-version ${RELEASE_VERSION:1} --channels candidate
```

---

It's true, thanks.